manifest/2.7.xml

```diff
@@ -14,14 +14,14 @@
     <!-- Build Scripts (required on CI servers) -->
     <project name="product-texts" path="product-texts" remote="couchbase"/>
     <project name="build" path="cbbuild" remote="couchbase">
-        <annotation name="VERSION" value="2.7.1"     keep="true"/>
+        <annotation name="VERSION" value="2.7.2"     keep="true"/>
```

Comment:
I think the build project should be pinned, based on the approach taken in 2.6.xml.

Reply:
Pinning to the current HEAD of cbbuild